recipe/meta.yaml

```diff
@@ -7,14 +7,19 @@ package:
 source:
   # We use the pypi URL as it includes the prepared Cython files.
   url: https://pypi.io/packages/source/p/pyzmq/pyzmq-{{ version }}.tar.gz
-  sha256: 824ad5888331aadeac772bce27e1c2fbcab82fade92edbd234542c4e12f0dca9
+  sha256: 6d18c76676771fd891ca8e0e68da0bbfb88e30129835c0ade748016adb3b6242
 
 build:
   number: 1
+  rpaths:
+    - lib/R/lib
   ignore_run_exports:
     - libsodium
     - libcxx          # [osx]
     - libstdcxx-ng    # [linux]
+    - zeromq          # [win]
+  missing_dso_whitelist:            # [win]
+    - $RPATH/libzmq.cp*-win32.pyd   # [win]
```

Comment:
Did something change such that we are no longer using the bundled ZeroMQ library on Windows? (See the comments below.)

Reply:
Added - zeromq # [not win] , but build still fails for missing_dso_whitelist: files. So kept those and build was successful